src/Components/Daterange.svelte

```diff
@@ -0,0 +1,422 @@
+  let shouldShakeDate = false;
+  let shakeHighlightTimeout;
+  let month = today.getMonth();
+  let year = today.getFullYear();
+
+  let isOpen = false;
+  let isClosing = false;
+
+  today.setHours(0, 0, 0, 0);
+
+  $: months = getMonths(start, end, selectableCallback, weekStart);
+
+  let monthIndex = 0;
+  $: {
+    monthIndex = 0;
+    for (let i = 0; i < months.length; i += 1) {
+      if (months[i].month === month && months[i].year === year) {
+        monthIndex = i;
+      }
+    }
+  }
+  $: visibleMonth = months[monthIndex];
+  $: visibleNextMonth = months[monthIndex + 1];
+
+  $: visibleMonthId = year + month / 100;
+  $: visibleNextMonthId = year + (month + 1) / 100;
+  $: lastVisibleDate = visibleNextMonth.weeks[visibleNextMonth.weeks.length - 1].days[6].date;
+  $: firstVisibleDate = visibleMonth.weeks[0].days[0].date;
+  $: canIncrementMonth = monthIndex + 1 < months.length - 1;
+  $: canDecrementMonth = monthIndex > 0;
+
+  export let formattedSelected;
+  export let formattedSelectedEnd;
+  $: {
+    if (typeof format === 'function') {
+      formattedSelected = format(selected);
+      formattedSelectedEnd = format(selectedEnd);
+    } else {
+      formattedSelected = formatDate(selected, format);
+      formattedSelectedEnd = formatDate(selectedEnd, format);
+    }
+  }
+
+  export let formattedCombined = '';
+
+  onMount(() => {
+    month = selected.getMonth();
+    year = selected.getFullYear();
+  });
+
+  function changeMonth(selectedMonth) {
+    month = selectedMonth;
+  }
+
+  function incrementMonth(direction, date) {
+    if (direction === 1 && !canIncrementMonth) return;
+    if (direction === -1 && !canDecrementMonth) return;
+    let current = new Date(year, month, 1);
+    current.setMonth(current.getMonth() + direction);
+    month = current.getMonth();
+    year = current.getFullYear();
+    highlighted = new Date(year, month, date || 1);
+  }
+
+  function getDefaultHighlighted() {
+    return new Date(selected);
+  }
+
+  function incrementDayHighlighted(amount) {
+    highlighted = new Date(highlighted);
+    highlighted.setDate(highlighted.getDate() + amount);
+    if (amount > 0 && highlighted > lastVisibleDate) {
+      return incrementMonth(1, highlighted.getDate());
+    }
+    if (amount < 0 && highlighted < firstVisibleDate) {
+      return incrementMonth(-1, highlighted.getDate());
+    }
+    return highlighted;
+  }
+
+  function getDay(m, date) {
+    for (let i = 0; i < m.weeks.length; i += 1) {
+      for (let j = 0; j < m.weeks[i].days.length; j += 1) {
+        if (areDatesEquivalent(m.weeks[i].days[j].date, date)) {
+          return m.weeks[i].days[j];
+        }
+      }
+    }
+    return null;
+  }
+
+  function checkIfVisibleDateIsSelectable(date) {
+    const dayThisMonth = getDay(visibleMonth, date);
+    const dayNextMonth = getDay(visibleNextMonth, date);
+    if (!dayThisMonth && !dayNextMonth) {
+      return false;
+    } if (!dayThisMonth && dayNextMonth) {
```

Comment:
```suggestion
    } 
    if (!dayThisMonth && dayNextMonth) {
```
```suggestion
    } else if (!dayThisMonth && dayNextMonth) {
```

Reply:
when I ran the `npm run lint -- --fix` it removed this for some reasons 🤷‍♂ This was what I had before